app/static/js/components/d3/Axis.jsx

```diff
@@ -1,80 +1,87 @@
-import React from 'react';
```

Comment:
@snallapa note to self. We will need to revisit in the case we need to ensure that there's more consistency with the themes.